pkg/util/util.go

```diff
@@ -91,3 +91,18 @@ func GetNextOrdinalPodName(podName string, ordinal int32) string {
 	basicStr := podName[:strings.LastIndex(podName, "-")]
 	return fmt.Sprintf("%s-%d", basicStr, ordinal+1)
 }
+
+func IsPodOutOfOrdinal(pod *corev1.Pod, expectedReplicas int32) (bool, error) {
```

Comment:
Seems this function checks if the pod ordinal exceeds replicas. Maybe you should add a comment about this function or rename this function.

Reply:
fix